test/dummy/app/commands/application_command.rb

```diff
@@ -14,4 +14,8 @@ class ApplicationCommand < TurboBoost::Commands::Command
   rescue_from TurboBoost::Commands::PerformError do |error|
     # do something...
   end
+
+  def resolve_state(changed_state)
+    state.merge! changed_state
```

Comment:
shouldn't that be `command_state.merge?` ?

Reply:
It's `state` when inside a command, and `command_state` when outside a command.